release-notes-8.0.0.md

```diff
@@ -4,9 +4,9 @@
 ${version-number}
 
 #### New Features
-- **US857114**: Introduced `CAF_RABBITMQ_PROTOCOL` environment variable so that RabbitMQ URL protocol is customisable.
-        This allows for TLS-enabled connections to be made to RabbitMQ if desired.
-        By default, this variable is set to "amqp" so there is no change in behaviour unless specified. 
+- **US857114: Introduced configurable Rabbit MQ protocol so that, if desired, Rabbit MQ communication can be TLS enabled. 
```

Comment:
```suggestion
- **US857114**: Introduced configurable Rabbit MQ protocol so that, if desired, Rabbit MQ communication can be TLS enabled. 
```

Reply:
Actually move this into breaking changes instead of the overlap explanation across new feature and breaking changes